### ISSUE TITLE
Outdated documentation about cargo dev bless 

### ISSUE THREAD

**Issue by kornelski** [rust-lang/rust-clippy#7151]
https://github.com/rust-lang/rust-clippy/blob/master/doc/adding_lints.md

The docs say that I need to run `cargo dev bless`, but when I do, I get:

```text
     Running `…/rust/target/debug/clippy_dev bless`
error: Found argument 'bless' which wasn't expected, or isn't valid in this context

USAGE:
    clippy_dev [SUBCOMMAND]

For more information try --help
```

And as of 28197b622611ba3a6367648974ccf59127c287bb there's no bless subcommand:

```
Clippy developer tooling

USAGE:
    clippy_dev [SUBCOMMAND]

FLAGS:
    -h, --help
            Prints help information

    -V, --version
            Prints version information


SUBCOMMANDS:
    fmt                    Run rustfmt on all projects and tests
    help                   Prints this message or the help of the given subcommand(s)
    limit_stderr_length    Ensures that stderr files do not grow longer than a certain amount of lines.
    new_lint               Create new lint and run `cargo dev update_lints`
    update_lints           Updates lint registration and information from the source code
```

**Comment by kornelski** [comment]
I've found that for blessing the tests this works:

```bash
./x.py test --stage 1 src/tools/clippy --bless
```

**Comment by jyn514** [comment]
That looks like a bug in the alias - bootstrap is just using `-- bless`.
https://github.com/rust-lang/rust/blob/master/src/bootstrap/test.rs#L654

**Comment by giraffate** [comment]
> https://github.com/rust-lang/rust-clippy/blob/master/doc/adding_lints.md
>
> The docs say that I need to run cargo dev bless, but when I do, I get:

It looks like it's resolved now, so I'm closing this.
